web/assets/js/views/PhysicalEvaluation-0.3-diversion-pouch.js

```diff
@@ -85,11 +85,11 @@ PMI.views['PhysicalEvaluation-0.3-diversion-pouch'] = Backbone.View.extend({
         }
         if (isWheelchairBound) {
```

Comment:
While we are fixing this, can we also change the `isWheelchairBound` variable to `isWheelchairUser`?